### PR TITLE
Santa tackles Bitcoin, part 1

### PR DESCRIPTION
**Pull request by bbrtj**
I thought I'll make it a two part article since it already is quite big and I don't want the second part to feel rushed. The second one should be shorter though.

#298

### CODE REVIEW

**Comment by oalders** [comment]
> I thought I'll make it a two part article

👍🏻

**Comment by bbrtj** [comment]
I have fixed the English to the best of my ability, but I'm sure it should still be looked at by a native speaker.

**Comment by oalders** [comment]
Closes #298

**Review by oalders — CHANGES_REQUESTED** [review]
This is really great. Thank you for writing this article. I know very little about Bitcoin and I found this to be a very interesting introduction. I just had a few comments. 

To address your comment above, I don't think we need to force every article into a native English speaker's voice as we'll lose some of the charm along the way.

What I'd like to do once these points have been addressed is merge the PR and then make a few small tweaks which should not detract from the intended meaning of the article. Once that is done, if you object to any of the changes we can certainly address that. That seems to me to offer a more streamlined approach than going back and forth on comma placement etc.

File: 2023/incoming/santa-tackles-bitcoin-part-one.pod [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+# generate a new random mnemonic with 256 bits of entropy
+my $mnemonic = generate_mnemonic(256);
+my $password = 'Christmas';
+
+# create a new master master key from the mnemonic
+my $master_key = btc_extprv->from_mnemonic($mnemonic, $password);
+
+# instruct the user to backup the mnemonic
+say "Your mnemonic is <$mnemonic>";
```
[the remark this answers is not in the document]
Comment (bbrtj) [reply 2023/incoming/santa-tackles-bitcoin-part-one.pod:102]: `say` is imported by `use v5.36;`, will a comment like the following one do?

`# (say function is a part of 5.36 version bundle)`

File: 2023/incoming/santa-tackles-bitcoin-part-one.pod [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+scheme|https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki> with
+L<purpose 84|https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki>.
+
+=begin perl
+
+# cuz Santa said so
+my $number_of_kids = 346;
+
+# get the master key for the BIP44 account
+# use purpose 84, which will create Segregated Witness addresses
```
[the remark this answers is not in the document]
Comment (bbrtj) [reply 2023/incoming/santa-tackles-bitcoin-part-one.pod]: Added a sentence of explanation with a link to wikipedia, thanks!

**Review by oalders — APPROVED** [review]

**Review by oalders — APPROVED** [review]
\o/

**Comment by oalders** [comment]
@bbrtj the edited article is at #303. I will merge and release shortly as it's scheduled for Dec 2. Let me know if there's anything you want to change, even after publication. I think it makes sense to run part 2 on Dec 3.

File: 2023/incoming/santa-tackles-bitcoin-part-one.pod [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+);
+
+# set replace-by-fee
+$tx->set_rbf;
+
+=end perl
+
+The transaction object was complete! Now all he had to do was to sign it and
+print it to the console. The transaction was not yet sent, but the serialized
+hex string can be broadcasted to the Bitcoin network at any time, since the
```
Comment (openstrike) [inline 2023/incoming/santa-tackles-bitcoin-part-one.pod:212]: s/broadcasted/broadcast/

File: 2023/incoming/santa-tackles-bitcoin-part-one.pod [hunk 4]
```diff
@@ -0,0 +1,280 @@
+Author: Bartosz Jarzyna <contact@bbrtj.eu>
+Title: Santa tackles Bitcoin, part one
+Topic: Bitcoin::Crypto
+
+=encoding utf8
+
+Santa is facing a new problem. Year after year, an increasing number of kids
+list Bitcoin as a thing they want to find under the Christmas Tree. This new
+trend comes much to displease of the Kriss Kringle.
```
[the remark this answers is not in the document]
Comment (bbrtj) [reply 2023/incoming/santa-tackles-bitcoin-part-one.pod:9]: Thanks, there were some changes done by Olaf before publishing, the word displease was replaced with displeasure. If this sentence isn't wrong then I'd like to keep it as it was published :)

File: 2023/incoming/santa-tackles-bitcoin-part-one.pod (hunk 3 above)
Comment (bbrtj) [reply 2023/incoming/santa-tackles-bitcoin-part-one.pod:212]: Fixed by Olaf

**Comment by oalders** [comment]
Thanks @openstrike! My bad for not letting you know about my edits. Was scrambling a bit to get that one out on time. 😅

**Pull request merged**